### PR TITLE
Update Presences: Mangalib, Ranobelib, Hentailib

### PR DESCRIPTION
**Pull request by Mirazex**
new website update = new presence update((

### CODE REVIEW

**Comment by Mirazex** [comment]
Proof:

InSa3qOtre
JlF6e66L71
orVYP4p0WK

**Review by Bas950 — CHANGES_REQUESTED** [review]
Bump versions

**Review by Alanexei — APPROVED** [review]

File: websites/H/Hentailib/dist/metadata.json [hunk 1]
```diff
@@ -9,7 +9,7 @@
     "en": "Website for reading hentai manga in Russian"
   },
   "url": "hentailib.me",
-  "version": "1.0.3",
+  "version": "1.0.4",
```
Comment (Bas950) [inline websites/H/Hentailib/dist/metadata.json:12]: Due to the amount of changes please bump to 1.1.0

File: websites/M/Mangalib/dist/metadata.json [hunk 2]
```diff
@@ -9,7 +9,7 @@
     "en": "Website for reading manga in Russian"
   },
   "url": "mangalib.me",
-  "version": "1.3.2",
+  "version": "1.3.3",
```
Comment (Bas950) [inline websites/M/Mangalib/dist/metadata.json:12]: Due to the amount of changes please bump to 1.4.0

File: websites/R/Ranobelib/dist/metadata.json [hunk 3]
```diff
@@ -9,7 +9,7 @@
     "en": "Website for reading ranobe in Russian"
   },
   "url": "ranobelib.me",
-  "version": "1.0.2",
+  "version": "1.0.3",
```
Comment (Bas950) [inline websites/R/Ranobelib/dist/metadata.json:12]: Due to the amount of changes please bump to 1.1.0

**Review by Bas950 — CHANGES_REQUESTED** [review]

**Review by Bas950 — APPROVED** [review]

**Review by Alanexei — APPROVED** [review]

**Pull request merged**